huntsman/utils/pyro/__init__.py

```diff
@@ -1,10 +1,101 @@
+import re
+import sys
+
 import Pyro4
 from Pyro4 import naming, errors
+from Pyro4.util import SerializerBase
+from astropy import units as u
+from astropy.io.misc import yaml as ayaml
+
+from pocs.utils import error
 
 from huntsman.camera.pyro import CameraServer
 from huntsman.utils import get_own_ip, sshfs, DummyLogger
 from huntsman.utils.config import load_device_config
 
+# Enable local display of remote tracebacks
+sys.excepthook = Pyro4.util.excepthook
+
+# Serialisers/deserialisers
+name_pattern = re.compile(r"error\.(\w+)'>$")
```

Comment:
`name_pattern` seems like an odd name for this variable. `error_pattern`?

Reply:
I guess I was thinking that this is the pattern that captures the name of the error from the string representation of its class, so `name_pattern`.  `error_pattern` is more descriptive, though.

Reply:
I feel like there must be a more elegant way to do what I'm using this regular expression for. I know Python has some powerful introspection features, so parsing `str(obj.__class__)` myself seems like something I shouldn't have to do. Do any alternatives spring to mind?